core/src/commonMain/kotlin/com/powersync/db/SqlCursor.kt

```diff
@@ -19,6 +19,13 @@ public interface SqlCursor {
     public val columnCount: Int
 
     public val columnNames: Map<String, Int>
+
+    public fun asMap(): Map<String, Any?> =
```

Comment:
It looks like the function signature mentions this will return a `Map<String, Any?>` while the implementation seems like the map values will always be of type `String?`. Perhaps we should document the behaviour of this method?

Reply:
Sorry, I've used that for debugging but it shouldn't be part of this PR. I've removed the method now.